tests/chaos/oadp/test_oadp.py

```
@@ -39,3 +40,38 @@ def test_reboot_vm_node_during_backup(
+            {
+                "vm_name": "vm-node-drain-12020",
+                "rhel_image": RHEL_LATEST["image_name"],
+            },
+            marks=pytest.mark.polarion("CNV-12020"),
+        ),
+    ],
+    indirect=True,
+)
+def test_drain_vm_node_during_backup(
+    oadp_backup_in_progress,
+    drain_vm_source_node,
```

Ruff (0.12.2)

Unused function argument: `drain_vm_source_node`

(ARG001)